.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
```

Comment:
If I understood correctly, the workflow only install dependencies and runs pytest but not any lint!

Reply:
Yep, it does the same as the Travis workflow. 
The sonar analysis will gather (linting) issues , but not make the job green.
It would be good idea to add a linter step to the job, but better to do that in another PR.